### PR TITLE
Update timeline

### PR DESCRIPTION
**Pull request by julieg18**
* remove extra time width on phone size
* take off extra margin/padding
* take off final time line
* add phone time line svg
* fix cursor in tablet/desktop

### CODE REVIEW

File: src/components/TimeLine/TimeLine.css [hunk 1]
```diff
@@ -1,11 +1,11 @@
 .time-line {
   overflow: hidden;
-  margin: 40px -16px 0;
```
Comment (iceicery) [inline src/components/TimeLine/TimeLine.css:3]: I am not sure we need to change this. Is the design changed? 
image

File: src/components/TimeLine/TimeLine.css [hunk 2]
```diff
@@ -112,11 +112,13 @@
...
 
   .time-line__time:hover {
-    cursor: default;
+    cursor: initial;
   }
 
   .time-line__line {
     left: calc(100% + 12px);
+    width: 133px;
+    background-image: url(../../images/timeline-line.svg);
```
Comment (iceicery) [inline src/components/TimeLine/TimeLine.css:121]: We probably need a timeline-line-tablet.svg, too. Lines are shown behind year numbers.
image

**Review by iceicery — CHANGES_REQUESTED** [review]
Nice job fixing the timeline. It looks great. Only two comments, please take a look.

File: src/components/TimeLine/TimeLine.css (hunk 1 above)
Comment (julieg18) [reply src/components/TimeLine/TimeLine.css:3]: We no longer need the the time line to stretch past the sides of the site, so no reason to add extra margin and padding.

File: src/components/TimeLine/TimeLine.css (hunk 2 above)
Comment (julieg18) [reply src/components/TimeLine/TimeLine.css:121]: I'll add a tablet SVG!

File: src/components/TimeLine/TimeLine.css (hunk 1 above)
Comment (iceicery) [reply src/components/TimeLine/TimeLine.css:3]: I see. I wonder if the map design is also changed: About Map Desktop: Reduce the map size (design oversight, TBD) #69. Does that mean we don't need to stretch our map past the sides of the site, either?

Comment (julieg18) [reply src/components/TimeLine/TimeLine.css:3]: TBD means to be decided. So we need to wait to see what the designer wants exactly for the map.

**Review by iceicery — CHANGES_REQUESTED** [review]
Code looks good. Did you add the svg file for tablet?

File: src/components/TimeLine/TimeLine.css [hunk 3]
```diff
@@ -118,7 +118,7 @@
   .time-line__line {
     left: calc(100% + 12px);
     width: 133px;
-    background-image: url(../../images/timeline-line.svg);
+    background-image: url(../../images/timeline-line-tablet.svg);
```
[the remark this answers is not in the document]
Comment (julieg18) [reply src/components/TimeLine/TimeLine.css:121]: Whoops, forgot to commit it 😅

**Review by iceicery — APPROVED** [review]
Nice job!

**Pull request merged**